content/blog/2022/q3-update/index.md

```diff
@@ -0,0 +1,95 @@
+
+**We grew our team**: We've hired two new team members to lead new major efforts with 2i2c. [**James Munroe**](https://2i2c.org/author/james-munroe/) will lead efforts around _community guidance and product design_, and [**Jim Colliander**](https://2i2c.org/author/jim-colliander/) will lead efforts around _partnerships and sustainability_. We also [updated our Hiring and Candidate Search documentation](https://github.com/2i2c-org/team-compass/issues/436) in the process.
+
+**We're refining our strategy**: We've begun a process of revisiting and refining our strategy after a year of major operations, see [our strategic update blog post for more information](../strategic-update/index.md).
+
+**We completed the [CSCCE](https://cscce.org) community management training**. Two of our team members (James and Sarah) both completed a several-week community management course that was offered in partnership with [CZI](chanzuckerberg.org).
+
+**Our team member Sarah began a part-time role as the JupyterHub Community Strategic Lead**. Sarah will be leading community strategy efforts within JupyterHub for the next two years thanks to a grant to the JupyterHub team from CZI. [Check out this issue to follow our progress](https://github.com/jupyterhub/team-compass/issues/536).
+
+
+<!-- 
+TODO: Only post this if we get our engineering salary bands questions resolved before this post goes live.
+
+**We are hiring an engineer**: Do you know an open source cloud engineer with experience in Kubernetes and has desire make research and education more impactful, accessible, and delightful? Check out our job ad [at ].
+-->
+
+## Service improvements
+
+We made a number of improvements to our cloud infrastructure and the processes around our service.
+Here's a brief breakdown:
+
+**We expanded our shared clusters to new cloud providers and regions**. We now have a shared clusters already deployed on Google Cloud Platform on `us-central1-b` and `europe-west2` and we will soon deploy a shared cluster on Amazon Web Services on `us-west-2`.
+
+**We defined an incident commander process**. This will allow us to coordinate and respond to major outages in our cloud infrastructure more efficiently. See [our incident response documentation](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/incidents.html) for more information.
+
+**We improved our cloud usage monitoring infrastructure**. We've deployed [a centralized Grafana Dashboard](https://github.com/2i2c-org/infrastructure/issues/328) that aggregates cloud usage across all of our partner communities, and allows us to keep track of any unexpected behavior or outages across them all. We're also looking into setting up automatic budgets and alerts for our infrastructure.
```

Comment:
Again, not sure about adding pieces we are currently working on (or planning to be working on).